File: Client/client/src/app/models/PersonaNatural/PersonaNaturalMainModel.ts
export class PersonaNaturalMainModel {
    constructor(
        public Item : number=0,
        public PersonaNaturalId: number = 0,
        public NomDocumento: string = '',
        public NumDocumento: string = '',
        public Nombres: string = '',
        public ApellidoPaterno: string = '',
        public ApellidoMaterno: string = '',
        public FechaRegistro: Date = new Date,
        public CodUsuario: string = ''
    ) { }
}

File: Desk/SpaceRT/ClientDesk/Model/PersonaNaturalMainModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ClientDesk.Model
{
    public class PersonaNaturalMainModel
    {
        public int Item { get; set; }
        public int PersonaNaturaId { get; set; }
        public String TipoDocumento { get; set; }
        public String NumDocumento { get; set; }
        public String Nombres { get; set; }
        public String ApellidoPaterno { get; set; }
        public String ApellidoMaterno { get; set; }
        public DateTime FechaRegistro { get; set; }
        public String CodUsuario { get; set; }
    }
}

